modules/getting-started-cli-scaling-app.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-cli-scaling-app_{context}"]
+= Scaling the application
+
+In Kubernetes, a `Deployment`(D) defines how an application deploys. In most cases, users use `Pod`, `Service`, `ReplicaSets`(RS), and `Deployment` resources together. In most cases, {product-title} creates the resources for you.
```

Comment:
(D) and (RS) are not acceptable shortened forms of these objects

Also, it's best to use "object" or "resource" after naming an object. The second sentence does it fine, but this first sentence should do "a `Deployment` object"